OpenBudgeteer.Blazor/Pages/Transaction.razor

```diff
@@ -183,8 +183,17 @@
                             </tr>
                         }
                     </table>
+                                    
+                    @if (transaction.Transaction.Amount != transaction.Buckets.Sum(b => b.Amount))
+                    {
+                        <div class="row" style="background-color: deepskyblue;  border-radius: 25px; margin: 0.3rem 0">
+                            <div class="col">Remaining</div>
```

Comment:
On thing that I have missed. What do you think in renaming this in something like "Difference" or something else as this value could be also negative (in case you distribute more money into the Buckets as the Amount of the Transaction)